erigon-lib/common/cmp/cmp.go

```diff
@@ -30,14 +30,3 @@ func InRange[T constraints.Ordered](min, max, val T) T {
 	}
 	return val
 }
-
-func Compare[T constraints.Ordered](a, b T) int {
```

Comment:
Why remove this?

Reply:
replaced with go's built in std `cmp` package - `cmp.Compare`